### ISSUE TITLE
[BUG] the split method in id is abcdef

### ISSUE THREAD

**Issue by charlie-bit** [graphprotocol/graph-network-analytics-subgraph#46]
I found the  split method in id is `abcdef` when call delegatorStakes query. but the subgraph in arb one network is spilt by `-`. please help unify it.

**Comment by juanmardefago** [comment]
Yes, that's part of the changes we made to move from `String` IDs to `Bytes` IDs, in order to improve performance and help it get it unstuck. 

Unfortunately we can't use `-` and had to figure out a pretty obvious separator to keep things consistent.

Ideally we should move the network subgraph to also use `Bytes`, and thus also use the `abcdef` separator, but big schema changes to the Core Network subgraph aren't something we should do lightly.
